pkg/artifact/artifacts_test.go

```diff
@@ -57,4 +59,39 @@ var _ = Describe("artifacts test", func() {
 			Expect(len(artifacts)).To(Equal(0))
 		})
 	})
+
+	Context("GetArtifactStream", func() {
+		It("should success", func() {
+			artifactClient.UserName = username
+			artifactClient.Token = password
+
+			projectName := "fakename"
+			pipelineName := "fakename"
+			filename := "fakename"
+			PrepareGetArtifact(roundTripper, artifactClient.URL, username, password, projectName, pipelineName, 1, filename)
+
+			body, err := artifactClient.GetArtifact(projectName, pipelineName, 1, filename)
+			Expect(err).To(BeNil())
+			Expect(func() bool {
```

Comment:
IMO, it would be better if we returned the body string.

Reply:
IMO, this might return a larger file. Using a body string can lead to a significant memory footprint.

Reply:
agree with @mzmuer , the artifact file might be a binary file. In that case, it is not possible to return a string.